.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tests/testdata"]
+	path = tests/testdata
+	url = git@github.com:openneuropet/testdata.git
```

Comment:
This line should be `url = https://github.com/openneuropet/testdata.git` Otherwise cloning the submodule fails from an environment without ssh access to the repository. This change should fix #124.